src/bci_build/package/templates/licensing_and_eula.j2

```diff
@@ -0,0 +1,22 @@
+## License
+SPDX-License-Identifier: {{ image.license }}
```

Comment:
Do we want this footer? It could be mistaken as the license of the whole image, which I doubt is the correct one here.

Reply:
it's what we currently declare as license on the container as label, why not have it in the readme?

Reply:
I do find the placement/naming ambiguous. Putting it without any comment into the Readme makes it look to me as a user like the license of the whole image. If you want to keep it, why not extending the Title to `License of the Dockerfile` or `License of the Build Recipe`?

Reply:
alright, I've added another paragraph on clarifying that licensing is complicated. how do you feel about the new version?